encoding/encoding.go

```diff
@@ -32,6 +32,20 @@ import (
 	"github.com/yarpc/yab/unmarshal"
 )
 
+// MethodType is type of the RPC method
+type MethodType int
+
+const (
+	// Unary method type is a traditional RPC
+	Unary MethodType = iota
+	// ClientStream method type RPC sends multiple messages
+	ClientStream
+	// ServerStream method type RPC receives multiple messages
+	ServerStream
+	// BidirectionalStream type RPC can send and receive multiple messages
+	BidirectionalStream
+)
```

Comment:
nit: to be consistent, let's use the same semantic as the yarpc now for the naming and comment. See https://github.com/yarpc/yarpc-go/blob/dev/api/transport/handler.go#L40

Reply:
The use-case here is quite different from YARPC, `MethodType` is verbose to identify different types of streaming RPCs as well which requires `client`, `server` and `bidirectional` stream